lib/resource.js

```diff
@@ -6,6 +6,7 @@ const {lstat, realpath, statify, normalisePath} = require("./utils.js");
 const MappedDisposable = require("mapped-disposable");
 const EntityType = require("./entity-type.js");
 const System = require("./system.js");
+const {Directory} = require("pathwatcher")
```

Comment:
There shouldn't be a need for the `pathwatcher` module, because the `Directory` class is already exposed as a member of `atom`. E.g., try running this in your dev-console:

~~~js
require("pathwatcher").Directory === require("atom").Directory;
~~~